frontend/src/components/project/disposal/ApprovalConfirmation.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { Form } from 'react-bootstrap';
+import { Field } from 'formik';
+
+export type OnChangeType = (checked: boolean) => void;
+
+interface IApprovalConfirmationProps {
+  onChange?: OnChangeType;
+  field: string;
+  checked?: boolean;
+  userDisplayName: string;
+}
+
+/**
+ * A component for displaying an approval confirmation checkbox
+ * @component
+ * @example
+ * const onChange = (checked) => console.log('confirmed');
+ * const userDisplayName = 'PIMS User';
+ * const fieldName = 'confirmation';
+ * return (
+ *
+ * <Formik initialValues={{[fieldName]: false}}>
+ *    <Form>
+ *        <ApprovalConfirmationCheckbox field={fieldName} onChange={onChange} userDisplayName={userDisplayName} />
+ *    </Form>
+ * </Formik>
+ * );
+ */
+export const ApprovalConfirmationCheckbox: React.FC<IApprovalConfirmationProps> = ({
```

Comment:
I wonder if this approval checkbox will need to be reused and could be more generic.  Don't do it now, but I suspect there will be more of them.

Reply:
It will be moved to the project disposal feature directory after Devin pulls in my code.